scripts/backfill_errors.py

```diff
@@ -0,0 +1,427 @@
+"""\
+Backfills the errors table from events.
+
+This script will eventually be moved to a migration - after we have a multistorage
```

Comment:
Since we only want to perform the backfill migration once, we can only start it when we are ready to begin filling the errors table. Otherwise we may be skipping events. The errors table is not being filled in dev, single tenant, onpremise - and is only being done for testing in SaaS (not the "proper" way with the multistorage consumer).